flexzpipe/FlexZPipe.py

```diff
@@ -25,26 +25,25 @@ class FlexZPipe(PipelineStage):
 #        ('photoz_pdfs', PhotozPDFFile),                                                   
         ('photoz_pdfs', HDFFile),]
     config_options = {
-        #"dz":   0.01,
-        #"zmin": 0.005,
-	#"zmax": 3.505,
         "chunk_rows": 1000,
         "bands": ["u","g","r","i","z","y"],
         "sigma_intrins": 0.05, #"intrinsic" assumed scatter, used in ODDS                  
         "odds_int": 0.99445, #number of sigma_intrins to integrate +/- around peak         
         # note that 1.95993 is the number of sigma you get for old "ODDS" =0.95            
-        #in old BPZ, 0.68 is 0.99445                                                       
-        #"point_estimate": "mode",  # mean, mode, or median          
+        #in old BPZ, 0.68 is 0.99445                                                           
         "has_redshift": True, #does the test file have redshift?
         #if so, read in and append to output file.
         "nz": 300, #Number of grid points that FZboost will calculate   
-        "model_picklefile": "/global/homes/c/chihway/FlexZPipe/flexcode_model_sqderr.pkl", #the pickle file containing the trained  flexzbooxt model.
+        "model_picklefile": "data/example/inputs/flexcode_model_sqderr.pkl", 
```

Comment:
The "flexcode_model_sqderr" filename was a temporary name during testing last week where I added 'objective':'reg:squarederror' to the regression_params dictionary in the config.yaml file (xgboost renamed the parameter 'reg:linear' to 'reg:squarederror' which can cause a ton of deprecation warnings if the parameter is not specified).   As the name is not super intuitive we may want to just rename the trained pickle model file back to flexcode_model.pkl.

Reply:
I've renamed this file.

Reply:
oops, looks like this hits the file size limit, so I can't push.  Will change it to be hosted at NERSC and add a note to download it.

Reply:
Okay, done now.